.github/workflows/issue_add.yml

```
@@ -10,6 +10,7 @@ permissions:
 
 jobs:
   add-to-project:
+    if: ${{ github.repository_owner == 'mattermost' }}
```

```suggestion
    if: ${{ github.repository_owner == 'mattermost' && github.event.pull_request.head.repo.fork == 'false' }}
```

Let's combine this with the fork event https://docs.github.com/en/webhooks/webhook-events-and-payloads#fork

---

Actually ignore this, I see that the trigger is not PR.